impeller/entity/entity_pass.cc

```diff
@@ -1200,6 +1191,16 @@ void EntityPass::SetEnableOffscreenCheckerboard(bool enabled) {
   enable_offscreen_debug_checkerboard_ = enabled;
 }
 
+int32_t EntityPass::GetChildrenRequiredMipCount() const {
+  int32_t result = 1;
+  for (auto& element : elements_) {
+    if (auto subpass = std::get_if<std::unique_ptr<EntityPass>>(&element)) {
+      result = std::max(result, subpass->get()->GetRequiredMipCount());
```

Comment:
If we're gonna keep `GetRequiredMipCount` around, we should just rename it to `GetBackdropFilterMipCount` since that's what it currently is.

Reply:
done